packages/py-api/py_api/controllers/hackathon/participants_controller.py

```diff
@@ -27,25 +25,17 @@ def get_all_participants(cls) -> JSONResponse:
                 status_code=404,
             )
 
-        # the dumps funtion from bson.json_util returns a string
+        # the dumps function from bson.json_util returns a string
         return JSONResponse(
             content={"participants": json.loads(dumps(participants))},
             status_code=200,
         )
 
     @classmethod
     def get_specified_participant(cls, object_id: str) -> JSONResponse:
-        try:
-            specified_participant = participants_col.find_one(
-                filter={"_id": ObjectId(object_id)},
-            )
-
-        except (InvalidId, TypeError) as e:
-            return JSONResponse(
-                content={"message": "Invalid object_id format!"},
```

Comment:
Why? This is a valid check.

Reply:
I have it defined as an app-level exception handler